### ISSUE TITLE
bug/PDF parsing failed, `parent_id` not in `old_to_new_mapping`

### ISSUE THREAD

**Issue by jieguangzhou** [Unstructured-IO/unstructured#2951]
**Describe the bug**
`parent_id` not in `old_to_new_mapping`

**To Reproduce**
```python
In [5]: from unstructured.partition.pdf import partition_pdf

In [6]: elements = partition_pdf("pdfs/Part_I/Endorsement/PerformanceSportsBrandsInc_20110909_S-1_EX-10.10_7220214_EX-10.10_Endorsement Agreement.pdf")
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Cell In[6], line 1
----> 1 elements = partition_pdf("pdfs/Part_I/Endorsement/PerformanceSportsBrandsInc_20110909_S-1_EX-10.10_7220214_EX-10.10_Endorsement Agreement.pdf")

File ~/workspace/SuperDuperDB/superduper-apps/legal_tech_demo/env/lib/python3.10/site-packages/unstructured/documents/elements.py:575, in process_metadata.<locals>.decorator.<locals>.wrapper(*args, **kwargs)
    573 unique_element_ids: bool = params.get("unique_element_ids", False)
    574 if unique_element_ids is False:
--> 575     elements = assign_and_map_hash_ids(elements)
    577 return elements

File ~/workspace/SuperDuperDB/superduper-apps/legal_tech_demo/env/lib/python3.10/site-packages/unstructured/documents/elements.py:523, in assign_and_map_hash_ids(elements)
    521     if not parent_id:
    522         continue
--> 523     e.metadata.parent_id = old_to_new_mapping[parent_id]
    525 return elements

KeyError: 'f904068ee17d49f32a4e9f06783bd523'

```

**Environment Info**
```
unstructured               0.13.3
unstructured-client        0.22.0
unstructured-inference     0.7.27
unstructured.pytesseract   0.3.12
```

**Additional context**
This mapping can currently be skipped via unique_element_ids=True

**Comment by jieguangzhou** [comment]
File:
[PerformanceSportsBrandsInc_20110909_S-1_EX-10.10_7220214_EX-10.10_Endorsement Agreement.pdf](https://github.com/Unstructured-IO/unstructured/files/15157191/PerformanceSportsBrandsInc_20110909_S-1_EX-10.10_7220214_EX-10.10_Endorsement.Agreement.pdf)

**Comment by scanny** [comment]
This was fixed in v0.13.5, if you upgrade `unstructured` to the latest that should do it :)
```bash
pip install -U unstructured
```

Closing as assumed resolved, but feel free to reopen if that doesn't do the trick :)

**Comment by jieguangzhou** [comment]
@scanny Thanks, It work now
